src/components/Garden/Home.js

```diff
@@ -210,6 +210,7 @@ const Home = function Home() {
                           onRequestNewProposal={handleRequestNewProposal}
```

Comment:
We could try to migrate this file to typescript. WDYT?

Reply:
Yah I'll love to do that, i just not do that because the file its huge and that PR need just small amout the code to fix. 

I tried now and need convert others files too. So idk if it worth for that PR.